listenbrainz/webserver/views/playlist_api.py

```diff
@@ -328,6 +328,51 @@ def create_playlist():
     return jsonify({'status': 'ok', 'playlist_mbid': playlist.mbid})
 
 
+@playlist_api_bp.route("/create-simple", methods=["POST", "OPTIONS"])
+@crossdomain(headers="Authorization, Content-Type")
+@ratelimit()
+@api_listenstore_needed
+def create_playlist_simple():
+    """
+    Create a quick and dirty public playlist. The POST data must be nothing but a list of recording_mbids
+    separated by , all on one line.
+
+    :reqheader Authorization: Token <user token>
+    :statuscode 200: playlist accepted.
+    :statuscode 400: invalid recording_mbid list sent
+    :statuscode 401: invalid authorization. See error message for details.
+    :statuscode 403: forbidden. The submitting user is not allowed to create playlists for other users.
+    :resheader Content-Type: *application/json*
+    """
+
+    user = validate_auth_header()
+
+    data = str(request.get_data(), "utf-8")
+    print(data)
+    try:
+        recording_mbids = data.split(",")
+        for mbid in recording_mbids:
+            uuid = UUID(mbid)
+    except ValueError:
+        raise APIBadRequest("Invalid data in recording mbid list. Should be nothing but recording_mbids separated by \\n.") 
+
+    playlist = WritablePlaylist(name="Instant playlist",
+                                creator_id=user["id"],
+                                description="Instant playlist -- nothing else is known. Can probably be deleted! :)",
+                                public=True)
+
+    for mbid in recording_mbids:
+        playlist.recordings.append(WritablePlaylistRecording(mbid=UUID(mbid), added_by_id=user["id"]))
+
+    try:
+        playlist = db_playlist.create(playlist)
+    except Exception as e:
```

Comment:
We have an error handler configured to capture all unhandled (50x) errors, give the users a specific message and report it to sentry. That error handler is going to run next anyways because APIInternalServerError is raised which is a 50x error so the try/except clause is redundant here.

We have a bunch of places in the codebase where such redundant clauses exists, I usually remove the ones when I am modifying related code and also haven't added those to new endpoints.

Reply:
Ah, ok.